packages/playground/README.md

```diff
@@ -0,0 +1,9 @@
+## RJSF Playground
+
+### Run local demo
+
+```bash
+npm install
+npm start
+```
+
```

Comment:
```suggestion
From the `packages/playground` directory:
```bash
npm start
` ` `
```

NOTE: There isn't an easy way to close the ` ```bash` block using the gitlab editor so that's what the three backticks spaced apart is trying to represent